src/assets.zig

```diff
@@ -15,8 +15,26 @@ var commonItems: std.StringHashMap(ZonElement) = undefined;
 var commonRecipes: std.StringHashMap(ZonElement) = undefined;
 var commonModels: std.StringHashMap([]const u8) = undefined;
 
+fn readDefaultFile(allocator: NeverFailingAllocator, dir: std.fs.Dir) !ZonElement {
+	if (dir.openFile("_defaults.zig.zon", .{})) |val| {
+		const string = try val.readToEndAlloc(main.stackAllocator.allocator, std.math.maxInt(usize));
+		defer main.stackAllocator.free(string);
+
+		return ZonElement.parseFromString(allocator, string);
+	} else |_| {}
```

Comment:
If the error is not `FileNotFound` it would make sense to return the error here.